e2e/pages/Onboarding3.po.js

```diff
@@ -0,0 +1,20 @@
+const buttonlabel = 'Next';
+const screenText =
+  'If you test positive, you can choose to donate your data anonymously';
+const screenshotText = 'Third Onboarding Page';
```

Comment:
nit: these come out unordered in filesystem, perhaps `Onboarding - Page 3` and so on